src/components/header/PracticeSection.tsx

```diff
@@ -10,74 +12,66 @@ const PracticeSection = ({ title }: { title: string }) => {
-            </div>
-            <div className="flex items-center border-4 absolute w-full top-[45%]">
-              <div className="h-[10px] w-full bg-[#dc2626] absolute"></div>
-              <div className="-right-4 absolute w-0 h-0 border-y-[15px] border-l-[30px] border-y-transparent border-l-[#dc2626] "></div>
-            </div>
-          </div>
-          <div className="flex justify-between items-start w-fit- gap-x-4-">
-            <p className="text-center text-xs max-w-52 leading-tight">
-              These methods result in design without a current contextual
-              understanding of the community, designing systems that reinforce
-              cultural tropes and knowledge hierarchies, or in short design, for
-              a theoretically imagined community.
-            </p>
-            <LevelCard
-              level={1}
-              title="Level 1"
-              description="Basic Principle"
-            />
-            <p className="text-center text-xs max-w-52 leading-tight">
-              Level 2 is where the methods of co-design work towards
-              inclusivity, understand FDPs as agents of change through
-              mobilising local resources and existing community-based structures
-              to support HE projects
-            </p>
-            <LevelCard
-              level={3}
-              title="Level 3"
-              description="Fighting the System"
-            />
+                      <p className="text-center text-xs h-32 leading-tight">
+                        {level.text}
+                      </p>
+                    </div>
+                  ) : (
+                    <div className="w-52">
+                      <p className="text-center text-xs h-32 leading-tight">
+                        {level.text}
+                      </p>
+                      <div className="relative">
+                        <div className="flex justify-center w-52">
+                          <Line />
+                        </div>
+                        <div className="flex items-center -pl-52 absolute w-full top-[45%]">
+                          <div className="h-[10px] w-52 bg-[#dc2626] absolute"></div>
+                          {index === length - 1 && (
+                            <div className="-right-4 absolute w-0 h-0 border-y-[15px] border-l-[30px] border-y-transparent border-l-[#dc2626] "></div>
+                          )}
+                        </div>
+                      </div>
+                      <div className=" h-32">
+                        <LevelCard
+                          level={level.level}
+                          title={level.title}
+                          description={level.description}
+                          attribute={level.attribute}
+                        />
+                      </div>
+                    </div>
+                  )}
+                </div>
+              </React.Fragment>
+            ))}
           </div>
```

Comment:
Can you put each part in its own component

Reply:
fixed